solidity/dashboard/src/stories/TransactionIsPendingMsgContent.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import TransactionIsPendingMsgContent from "../components/ViewTxMsgContent"
+import centered from "@storybook/addon-centered/react"
+
+const mockedTransactionHash =
+  "6146ccf6a66d994f7c363db875e31ca35581450a4bf6d3be6cc9ac79233a69d0"
+
+export default {
+  title: "TransactionIsPendingMsgContent",
+  component: TransactionIsPendingMsgContent,
```

Comment:
This component is no longer used- so I think we can remove this story and get rid of this component from the dapp.